Projet/render/GameFactory.cpp

```diff
@@ -77,7 +77,13 @@ void GameFactory::CreatePlayer(Math::Transform Transform)
 	auto PlayerBody = MyPlayer->AddComponent<RigidBody>(RigidBody::RigidActorType::Dynamic);
 	PlayerBody->SetMass(10.f);
 
+	PM3D::CMoteurWindows::GetInstance().SetSkyBox(new Skybox{
+	&MyPlayer->Transform, PM3D::CMoteurWindows::GetInstance().GetResourcesManager().GetMesh(L".\\modeles\\sky\\sky.OMB"),
+	PM3D::CMoteurWindows::GetInstance().GetResourcesManager().GetShader(L".\\shaders\\MiniPhong.fx") });
```

Comment:
Why get a shader ?

Reply:
to render the texture and the lights, the skybox is a obj